modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/internal/CompactedTopologyHistorySelfTest.java

```diff
@@ -231,8 +233,10 @@ public void testNodeChangeTheirAttributes() throws IgniteCheckedException {
 
         CompactedTopologyHistory compacted = new CompactedTopologyHistory(history);
 
-        byte[] bytes = JdkMarshaller.DEFAULT.marshal(compacted);
-        compacted = JdkMarshaller.DEFAULT.unmarshal(bytes, Thread.currentThread().getContextClassLoader());
+        JdkMarshaller marshaller = new JdkMarshaller();
```

Comment:
TEST_JDK_MARSHALLER

Reply:
This class is not a subclass of GridCommonAbstractTest.